dist/index.js

```diff
@@ -111,7 +111,9 @@ exports.default = Bookshelf => {
         // Parse JSON columns.
         Object.keys(attributes).forEach(attribute => {
           if (this.constructor.jsonColumns.indexOf(attribute) !== -1) {
-            model.attributes[attribute] = JSON.parse(model.attributes[attribute]);
```

Comment:
All good, but please remove changes on this file as the `dist` files are only added on the release commit.